README.md

```diff
@@ -9,21 +9,25 @@ for testing purposes.
 1. Clone this repo
    `git clone https://github.com/mailsac/mailsac-examples.git`
 
-2. Install sphinx
-   `pip install Sphinx sphinx_rtd_theme sphinx-tabs`
-
-3. Change directories to the cloned repo
+2. Change directories to the cloned repo
    `cd mailsac-examples`
 
-4. Run build script
+3. Setup virtual environment (option but helps isolate python modules)
+   `python3 -m venv venv`
```

Comment:
Just double checking - two `venv`?

Reply:
the first `venv` is the python module, the second is the parameter to create a virtual environment in the directory `venv`